pkg/api/v1/atlasproject_types.go

```diff
@@ -125,28 +125,10 @@ const hiddenField = "*** redacted ***"
 //nolint:errcheck
 func (p AtlasProjectSpec) MarshalLogObject(e zapcore.ObjectEncoder) error {
 	printable := p.DeepCopy()
-	// cleanup encryption at EncryptionAtRest
-	if printable.EncryptionAtRest != nil {
-		printable.EncryptionAtRest.AwsKms.AccessKeyID = hiddenField
-		printable.EncryptionAtRest.AwsKms.CustomerMasterKeyID = hiddenField
-		printable.EncryptionAtRest.AwsKms.SecretAccessKey = hiddenField
-		printable.EncryptionAtRest.AwsKms.RoleID = hiddenField
-		printable.EncryptionAtRest.AzureKeyVault.Secret = hiddenField
-		printable.EncryptionAtRest.GoogleCloudKms.ServiceAccountKey = hiddenField
-	}
-
```

Comment:
I know you made some of the fields unexported, but aren't they still going to be printed?

Reply:
I'm fairly sure that [Zap does not print unexported fields](https://github.com/uber-go/zap/issues/1097#issuecomment-1124393884) - I think because it's built around JSON marshalling. Just to be safe, I've set the printable fields to hiddenField as it was before.